src/wallet/wallet.h

```diff
@@ -88,6 +88,10 @@ enum WalletFeature
     FEATURE_COMPRPUBKEY = 60000, // compressed public keys
 
     FEATURE_HD = 130000, // Hierarchical key derivation after BIP32 (HD Wallet)
+
+    //TODO: FEATURE_HD_SPLIT needs to be bumped to 140000 once we branch of 0.14 //
```

Comment:
I'm completely confused by this? why would the version of the wallet change due to a branch here? won't that break compatibility with wallets generated by git master and cause funds loss for them?